src/components/modal/AposModalFooter.vue

```diff
@@ -1,17 +0,0 @@
-<template>
```

Comment:
This component doesn't add much. There's no reason I see not to have it directly in AposModal instead.